From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 24 Feb 2017 21:06:23 +0100
Subject: [PATCH] {MAYBE UPS} hv: silence -Wunused-label
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes: dee863b571b0 ("hv: export current Hyper-V clocksource")

x86-32 gcc warning:

    arch/x86/hyperv/hv_init.c:167:1: error: label ‘register_msr_cs’
    defined but not used [-Werror=unused-label]
---
 arch/x86/hyperv/hv_init.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/hyperv/hv_init.c b/arch/x86/hyperv/hv_init.c
index db64baf0e500..e868d5d2567d 100644
--- a/arch/x86/hyperv/hv_init.c
+++ b/arch/x86/hyperv/hv_init.c
@@ -164,7 +164,9 @@ void hyperv_init(void)
 	 * the partition counter.
 	 */
 
+#ifdef CONFIG_X86_64
 register_msr_cs:
+#endif
 	hyperv_cs = &hyperv_cs_msr;
 	if (ms_hyperv.features & HV_X64_MSR_TIME_REF_COUNT_AVAILABLE)
 		clocksource_register_hz(&hyperv_cs_msr, NSEC_PER_SEC/100);
-- 
